src/main/java/edu/umich/tl/CourseDelete.java

```diff
@@ -75,6 +76,7 @@
 	private static  String emailSubjectText="Report from canvas course delete utility for terms %s";
 	
 	private static Log M_log = LogFactory.getLog(CourseDelete.class);
+	private static boolean isCourseDeleteEnabled=false;
```

Comment:
This symbol should probably be named "courseDeletionEnabled".  Usually symbols that begin with "is" are getter methods that return Boolean values.  For example, "isCourseDeletionEnabled()" would return the value of "courseDeletionEnabled".